web/modules/custom/joinup_migrate/src/Plugin/migrate/source/JoinupSqlBase.php

```diff
@@ -72,12 +73,18 @@ protected function getLegacySiteFiles() {
    * {@inheritdoc}
    */
   public function prepareRow(Row $row) {
-    if ($this->reservedUriTables && $row->hasSourceProperty('uri') && ($uri = $row->getSourceProperty('uri'))) {
-      $reserved = $this->getUrisToExclude();
-      if (in_array($uri, $reserved)) {
-        // This URI is in the reserved list. Generate a new one.
+    if ($row->hasSourceProperty('uri') && ($uri = $row->getSourceProperty('uri'))) {
+      if (Unicode::strlen($uri) > 255) {
```

Comment:
Shouldn't we log this? This will break the import of the data of federated repos in the future.